Gem/Code/Source/Automation/PrecommitWizardSettings.h

```diff
@@ -9,27 +9,32 @@
 #include <Automation/ScriptReporter.h>
 #include <AzCore/std/containers/vector.h>
 #include <AzCore/std/containers/map.h>
+#include <AzCore/Preprocessor/Enum.h>
 
 namespace AtomSampleViewer
 {
     struct PrecommitWizardSettings
     {
         
-        enum Stage
+        enum class Stage
         {
             INTRO,
             REPORT_FULLSUITE_SUMMARY,
             MANUAL_INSPECTION,
             REPORT_FINAL_SUMMARY
         };
 
-        enum ImageDifferenceLevel {
-            NO_DIFFERENCE = 0,
-            LOW_DIFFERENCE = 1,
-            MODERATE_DIFFERENCE = 2,
-            HIGH_DIFFERENCE = 3,
+        struct ImageDifferenceLevel
+        {
+            enum Levels
```

Comment:
I think "enum class ImageDifferenceLevel" would give you basically the same thing.

Reply:
enum class won't allow implicit conversion to int which is why i opted for this.

Reply:
Where do we need implicit conversion? Can you just do a numeric cast?